.vib/owncloud/cypress/cypress/integration/owncloud_spec.js

```diff
@@ -4,77 +4,36 @@ import { random } from '../support/utils';
 it('allows creating a folder and uploading a file ', () => {
```

Comment:
If the test does also download the file, please update the description string accordingly